src/ol_infrastructure/applications/ocw_studio/__main__.py

```diff
@@ -196,4 +201,54 @@
         secret=vault_secrets["github_shared_secret"],
     ),
 )
+
+# Setup AWS MediaConvert Queue
+ocw_studio_mediaconvert_queue = mediaconvert.Queue(
+    "ocw-studio-mediaconvert-queue",
+    description="OCW Studio Queue",
+    name=f"ocw-studio-mediaconvert-queue-{stack_info.env_suffix}",
+    tags=aws_config.tags,
+)
+
+# Configure SNS Topic and Subscription
+ocw_studio_sns_topic = sns.Topic(
+    f"ocw-studio-sns-{stack_info.env_suffix}-topic", tags=aws_config.tags
+)
+
+ocw_studio_sns_topic_subscription = sns.TopicSubscription(
+    "ocw-studio-sns-topic-subscription",
+    endpoint="https://{}/api/transcode-jobs/".format(
+        ocw_studio_config.require("app_domain")
+    ),
+    protocol="https",
+    topic=ocw_studio_sns_topic.arn,
+)
+
+# Configure Cloudwatch EventRule and EventTarget
+ocw_studio_mediaconvert_cloudwatch_rule = cloudwatch.EventRule(
+    "ocw-studio-mediaconvert-cloudwatch-eventrule",
+    description="Capture MediaConvert Events for use with OCW Studio",
+    event_pattern="""{
```

Comment:
Any reason not to do a `json.dumps` of the structure rather than using a triple-quoted string?